app/components/primer/beta/breadcrumbs.pcss

```diff
@@ -27,4 +27,9 @@
 
 .breadcrumb-item-selected a {
   color: var(--fgColor-default);
+
+  &:hover, &:focus {
+    cursor: default;
+    text-decoration: none;
+  }
 }
```

Comment:
```suggestion
.breadcrumb-item-selected a {
  color: var(--fgColor-default);
  cursor: default;
  text-decoration: none;
}
```

I believe this will fix that strange issue

Reply:
Oh, yay. This is great. I'll add it now